clippy.toml

```diff
@@ -0,0 +1,56 @@
+msrv = "1.67"
+
+
+# https://rust-lang.github.io/rust-clippy/master/index.html#disallowed_macros
+disallowed-macros = [
+  'dbg',
+
+  # TODO(emilk): consider forbidding these to encourage the use of proper log stream, and then explicitly allow legitimate uses
```

Comment:
what's keeping us from that so far? Too widespread direct use?

Reply:
Yeah - it is used in a lot of build scripts and tests. We can add whitelists to those files but I didn't want to make this PR too big or controversial in one go